Hearthstone Deck Tracker/Controls/DeckImportingControl.xaml.cs

```diff
@@ -112,6 +112,31 @@ private void CheckBoxImportAll_OnModified(object sender, RoutedEventArgs e)
 			}
 		}
 
+		private void CheckBoxImport_OnChecked(object sender, RoutedEventArgs e)
+		{
+			int modifiedCount = 0;
+
+			if (ItemsControl == null)
+				return;
+
+			foreach(var item in ItemsControl.Items)
+			{
+				var c = (ContentPresenter)ItemsControl.ItemContainerGenerator.ContainerFromItem(item);
+				var chk = (CheckBox)c.ContentTemplate.FindName("CheckBoxImport", c);

```

Comment:
Since you are using this in OnModified as well, `c` and `chk` can be extracted to a `GetCheckBox` method, making both methods a bit cleaner.